sonar-project.properties

```diff
@@ -0,0 +1,10 @@
+sonar.projectKey=fh-logger
+sonar.projectName=fh-logger-nightly-master
+sonar.projectVersion=0.4.1
+
+sonar.sources=./lib
+sonar.tests=./test
+sonar.language=js
+sonar.javascript.lcov.reportPath=./coverage/lcov.info
```

Comment:
General note here: We can of course investigate and make relevant JIRAs one we have the builds consistent. The above config file is a template for the core configuration that should be included so the coverage path could be wrong (again a general note). 

In the above case the file is present it is just empty. There is a cobertura report there however.Something to research further down the line as to why it's not generating or if we can use an alternative.